src/utils/ipfs/restIpfsNode.ts

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+
+export class RestIpfsNode {
+  readonly node_url: string;
```

Comment:
will we use classes?

Reply:
In such case, I can suppose that we can use multiple ipfs nodes, is good to have some abstraction levels